docs/modules/ROOT/pages/references/crds.adoc

```diff
@@ -797,6 +778,8 @@ VSHNPostgreSQL is the API for creating Postgresql clusters.
 | Field | Description
 | *`schedule`* __string__ | 
 | *`retention`* __integer__ | 
+| *`deletionProtection`* __boolean__ | DeletionProtection will protect the instance from being deleted for the given retention time. This is enabled by default.
+| *`deletionRetention`* __integer__ | DeletionRetention specifies in days how long the instance should be kept after deletion. The default is keeping it one week.
```

Comment:
```suggestion
| *`deletionRetention`* __integer__ | DeletionRetention specifies in days how long the instance should be kept after deletion. The default value is one week.
```

Reply:
This is a completely unrelated change and is auto generated. I won't change this in this PR